File: app/src/main/java/com/example/myers/diff/DiffNode.java
package com.example.myers.diff;

/**
 * @author gk
 * @Description:
 * @date 2023/12/25
 */
public final class DiffNode extends PathNode {
    public DiffNode(int i, int j, PathNode prev) {
        super(i, j, prev);
    }

    @Override
    public Boolean isSnake() {
        return false;
    }
}

File: app/src/main/java/com/example/myers/diff/DiffNode.kt
package com.example.myers.diff

/**
 * @author gk
 * @Description:
 * @date 2023/12/25
 */
class DiffNode(i: Int, j: Int, prev: PathNode?) : PathNode(i, j, prev) {
    override val isSnake: Boolean
        get() = false
}